pkg/server/commands/reverseexpand/reverse_expand.go

```diff
@@ -167,11 +167,16 @@ type ReverseExpandResult struct {
 
 type ResolutionMetadata struct {
 	QueryCount *uint32
+
+	// The number of times we are expanding from each node to find set of objects
+	// In case of intersection and exclusion, it also includes the dispatch counts done for Check calls to exclude Objects
+	DispatchCount *uint32
```

Comment:
ReverseExpand code is unaware that Checks are happening, that happens outside, in ListObjects.
```suggestion
	// The number of times we are expanding from each node to find set of objects
	DispatchCount *uint32
```